src/app/modules/admin/components/team-cards/team-cards.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="team-cards">
+  <app-team-card
+    *ngFor="let teamCard of teamCards"
+    [teamCard]="teamCard"
+  ></app-team-card>
+</div>
```

Comment:
Объединить всё в один компонент. Компонент не такой большой, чтобы разбивать его на 2.
```suggestion
<div class="team-cards">
  <ng-container
    *ngFor="let teamCard of teamCards"
  >
  /* твой код из team-card */
  </ng-container>
</div>
```